### ISSUE TITLE
Managing logrotate

### ISSUE THREAD

**Issue by carlalexander** [debops/ansible-nginx#114]
I need to change logrotate to rotate the nginx logs daily. I know I can do that with `debops.logrotate`. That said, does it make sense to have the role manage its own logrotate configuration?

**Comment by drybjed** [comment]
Definitely! `debops.logrotate` is designed to being used as a role dependency, evident by `logrotate__dependent_config` variable. It's just that the role is new, and no other role has configuration for it yet.

If you remember to divert the stock `/etc/logrotate.d/*` configuration files by adding `item.divert: True` to the configuration, you can easily replace the default configuration with your own.

**Comment by carlalexander** [comment]
Yeah, I'm doing that right now. I was just wondering if we should add that to `debops.nginx`?

**Comment by drybjed** [comment]
I suppose we could... You mean the logs in `/var/log/nginx/`, right?

**Comment by carlalexander** [comment]
Yep
